Cbk13 sitzplatzbuchung

hannah1666 opened the pull request:
Farbe in DateTimeSelection wird geregelt - evtl noch Anpassungen, aber an sich funktioniert

hannah1666 committed:
adds Icon + checks if not Sat or Sun

hannah1666 committed:
adds comment: calculate the color

ich fang mal an, alles was ich im Code verstehe und brauche zu kommentieren, damit ichs nicht jedes mal wieder erneut nachvollziehen muss - hoffe das ist okay!

hannah1666 committed:
Kommentare + Versuch Farben anzugeben

Commit:
fixed bug on pass params for route

hannah1666 committed:
fixes bug

Commit:
saves temp variables in storage

hannah1666 committed:
bug fix - loading vor Arbeitsplatzauswahl eingefügt

hannah1666 committed:
Start: Ausgabe this.bookings

hannah1666 committed:
comments on various functions

hannah1666 committed:
comments & inserts Color calculation in DayPicker

thelinking committed:
Button zur Platzbuchung

- Buttondesign eingefügt
- "Platz reservieren" -> "Platz buchen"

thelinking committed:
Buttondesign angepasst

- Code kommentiert
- Abstand zwischen Text und Button eingefügt
- Farbe des Buttons
- Textgröße

hannah1666 committed:
calculates Farbanzeige Daypicker depends on capacity

wall0404 commented on resources/js/components/Pages/Book/Page_DateTimeSelection.vue:
```diff
@@ -31,7 +32,19 @@ export default {
+        else
+            localStorage.setItem("bookings_" + this.$route.params.workstation_id, JSON.stringify(this.bookings));
+
+console.log(this.bookings)
```
Bitte noch raus nehmen. Keine Test-Ausgaben auf dem Server :)

hannah1666 replied:
erledigt :)

wall0404 commented on resources/js/components/Elements/DayPicker.vue:
```diff
@@ -64,8 +64,25 @@ export default {
+                    }
+                    if(color === "red") {
+                        disabled = true;
+                        console.log(disabled);
```
Bitte noch raus nehmen. Keine Test-Ausgaben auf dem Server :)

hannah1666 committed:
deletes test output

hannah1666 committed:
entfernt test ausgabe agaiin

hannah1666 committed:
Update DayPicker.vue

hannah1666 committed:
important

hannah1666 committed:
important eingefügt

Commit:
adds error handling in api for booking

hannah1666 committed:
merged develop

hannah1666 committed:
Merge branch 'CBK13-Sitzplatzbuchung-API' into CBK13-Sitzplatzbuchung

mira853 committed:
Abgelaufenen Token refreshen

thelinking committed:
Button rechtsbündig

thelinking committed:
Kommentar

thelinking committed:
Merge branch 'CBK8-Homescreen' into CBK13-Sitzplatzbuchung

hannah1666 committed:
modifies seat view so that colors are displayed and adjusts formatting

hannah1666 committed:
Merge branch 'CBK13-Sitzplatzbuchung' of https://github.com/wall0404/coba into CBK13-Sitzplatzbuchung

hannah1666 committed:
changes amount of hours when the indicator gets red & names of the color inidcators

thelinking committed:
Icon zu "Platz buchen"-Button eingefügt

hannah1666 committed:
Anpassungen Sitzplatzansicht Liste

- Namensänderung Sitzplätze
- Locationname-Angabe in Header
- Sitzplatzname unter Button
- Abstandsanpassung

hannah1666 committed:
Merge branch 'CBK13-Sitzplatzbuchung' of https://github.com/wall0404/coba into CBK13-Sitzplatzbuchung

hannah1666 committed:
Farbanpassung disabled day

mira853 committed:
die Navbar wurde aus Sign UP/In entfernt, dafür Buttons hinzugefügt und die es wurden Details angepasst

Commit:
adss front end slider check if already booked

Commit:
removes half an hour puffer between to bookings

mira853 committed:
Das Aussehen vom Coba button wurde angepasst

Eine Klasse wurde dafür hinzugefügt

hannah1666 committed:
Merge branch 'origin/CBK9-SignIn' into CBK13-Sitzplatzbuchung

hannah1666 committed:
Info Button beiw siwtzplatzauswahl + modalöffnung mit iinfos + triwggerbox

wall0404 commented:
Wird zusammen mit #35 gemerged
